.github/workflows/ci.yml

```diff
@@ -106,3 +106,18 @@ jobs:
           HUSKY: 0
           NPM_TOKEN: ${{secrets.NPM_TOKEN}}
           GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
+  chromatic-deployment:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3.1.0
+        with:
+          fetch-depth: 0
+      - uses: pnpm/action-setup@v2.2.3
+      - name: Install dependencies
+        run: pnpm install
+      - name: Publish to Chromatic
+        if: github.ref != 'refs/heads/main'
```

Comment:
why not on main ?

Reply:
@chambo-e My bad I tough we will still stay on our infa for deploy